src/.vuepress/theme/styles/layout/_icontable.scss

```diff
@@ -0,0 +1,60 @@
+.IconTable {
+  width: 100%;
+  padding: 1em;
+
+  &-item {
+    display: inline-block;
+    width: 50%;
+    height: 6em;
+    border-left: 1px solid #e1e4e5;
+    border-bottom: 1px solid #e1e4e5;
+    padding: 1em;
+    background-color: #eee;
+
+    &:nth-child(-n + 2) {
+      border-top: 1px solid #e1e4e5;
+    }
+
+    &:nth-child(even) {
+      border-right: 1px solid #e1e4e5;
+    }
+
+    &:nth-child(4n),
+    &:nth-child(4n-1) {
+      background-color: white;
+    }
+  }
+
+  &-item-icon {
+    display: table-cell;
+    vertical-align: middle;
+    width: 60px;
+    height: 60px;
+
+    img {
+      width: 100%;
+    }
+  }
+
+  &-item-text {
+    display: table-cell;
+    vertical-align: middle;
+    padding-left: 1em;
+    overflow: hidden;
+    width: calc(100% - 5em);
+    height: calc(100% - 2em);
+  }
+}
+
+@media (max-width: 720px) {
```

Comment:
here, you can use the breakpoints declared [here](https://github.com/kuzzleio/documentation/blob/master/src/.vuepress/theme/styles/_config.scss#L16)

Reply:
TAHT IZ UR KOD BRUH (I just copy pasted it from the component 😂 )